CONTRIBUTING.md

```diff
@@ -0,0 +1,122 @@
+# Introduction
+
+Let's start with the relationship between several related projects:
+
+* **Kubernetes** is a container orchestration system, and is used to run
+  containers on a cluster
+* **containerized-data-importer (CDI)** is an add-on which solves the problem of
+  populating Kubernetes Persistent Volumes with data.  It was written to be
+  general purpose but with the virtualization use case in mind.  Therefore, it
+  has a close relationship and special integration with KubeVirt.
+* **KubeVirt** is an add-on which is installed on-top of Kubernetes, to be able
+  to add basic virtualization functionality to Kubernetes.
+
```

Comment:
The above break down is a nice touch IMO, should emphasize it in the README as well probably; currently the two don't match :)